### ISSUE TITLE
Invite

### ISSUE THREAD

**Issue by sarahmonster** [sarahmonster/save-me-the-waltz#13]
- [ ] outer envelope (plain, doesn't need to be designed)

- [ ] info re: donating to Boys and Girls club (business card size)
- [ ] RSVP card + envelope (5"x3.5), envelope printed w/ address (a1 white envelope)
- [ ] programme/information card (slightly bigger)
- [ ] actual invite card (should fold open and contain other bits & bobs, held together with custom initial sticker, postcard size 5x7)

paper stock: ultra cardstock (130lb uncoated cover)

send out by June 1. Design ready by April 1. @jennybeans to provide content.

**Comment by jennybeans** [comment]
invite text:
Jennifer Lea Arnold & Richard Lingxiang Li
invite you to celebrate their marriage
Saturday, August 19, 2017 at 5:00pm
NeXT restaurant: 6400 Hazeldean Road
Dinner & frivolity to follow

**Comment by sarahmonster** [comment]
For some reason my brain is totally blocked on this one, so I'm urging it to start firing again. Here's a super-rough mockup:

<img width="379" alt="mock-a" src="https://cloud.githubusercontent.com/assets/376315/25067952/f34ce26e-224a-11e7-9551-2f2e49d9921c.png">

I'm trying to focus on sorting out structure first, then going from there, so let's close our eyes and go "la la la!" about it looking a bit meh right now and focus on the size/structure/placement bits. I can refine the visuals once that lot is all set.

We talked about doing a postcard-sized invitation card that would open up and have all the other bits inside, which I think would be really nice. Is this still the plan? Can we do this with catprint? (Their website confused me.) If it's going to fold open, do we want a horizontal cut or a vertical cut? (ie which way should the "doors" open?) Should we have text on the outside as well? Are we still going to look into closing it with an initial sticker?

SO MANY QUESTIONS.

**Comment by jennybeans** [comment]
So I think perhaps that I was looking for something a bit different from the STDs  for the invite itself? It's totally okay if you don't have the time to work on these - let me know and I'll totally understand! I know you have a lot going on & really appreciate all the work you've done.

**Comment by sarahmonster** [comment]
Okay! I thought you'd want it to match the STDs/website, but we can go different if you'd prefer. Are there elements (colour, typography, graphic elements) you'd like to carry over at all, or would you like to go for something totally new? Stylistically, did you still want to do an art deco theme, or would you like to try something else? Do you have some examples of design styles you like?

**Comment by jennybeans** [comment]
No, I totally understand! I love the STDs and the website, so something similar is definitely what I'm looking for without it being the exact same, maybe? I don't design so I know it's a challenge.On Apr 16, 2017 7:05 AM, sarah ✈ semark <notifications@github.com> wrote:Okay! I thought you'd want it to match the STDs/website, but we can go different if you'd prefer. Are there elements (colour, typography, graphic elements) you'd like to carry over at all, or would you like to go for something totally new? Stylistically, did you still want to do an art deco theme, or would you like to try something else? Do you have some examples of design styles you like?

**Comment by sarahmonster** [comment]
Okay, I have some thoughts here I'm going to percolate a bit while we figure out the structure/size bit. Did you want to do a card that opens up (like doors?) and can catprint make that sort of thing happen?

**Comment by jennybeans** [comment]
I think it's probably easiest to have a set of flat cards! I can ribbon-tie them if I feel really fancy. Thank you!On Apr 16, 2017 8:13 AM, sarah ✈ semark <notifications@github.com> wrote:Okay, I have some thoughts here I'm going to percolate a bit while we figure out the structure/size bit. Did you want to do a card that opens up (like doors?) and can catprint make that sort of thing happen?

**Comment by sarahmonster** [comment]
Sure thing. How about if we go horizontal instead of vertical? I could put your names side by side that way...

**Comment by jennybeans** [comment]
That sounds lovely!On Apr 16, 2017 9:09 AM, sarah ✈ semark <notifications@github.com> wrote:Sure thing. How about if we go horizontal instead of vertical? I could put your names side by side that way...

**Comment by sarahmonster** [comment]
Things we need copy for:

 - info re: donating to Boys and Girls club (business card size)
- RSVP card + envelope (5"x3.5), envelope printed w/ address (a1 white envelope)
- programme/information card (slightly bigger)

Also I think we aren't doing the initial sticker anymore, since the invite doesn't open? But let me know if that's not the case!

**Comment by sarahmonster** [comment]
Proof for content/size/etc!

<img width="994" alt="screenshot 2017-04-27 14 22 06" src="https://cloud.githubusercontent.com/assets/376315/25485301/63dcf25e-2b55-11e7-8b5a-2cc0830ab9c0.png">
